app/models/sales_log.rb

```diff
@@ -34,7 +34,7 @@ class SalesLog < Log
   scope :search_by, ->(param) { filter_by_id(param) }
   scope :filter_by_organisation, ->(org, _user = nil) { where(owning_organisation: org) }
 
-  OPTIONAL_FIELDS = %w[purchid].freeze
+  OPTIONAL_FIELDS = %w[purchid monthly_charges_value_check].freeze
```

Comment:
I'm interested what this does.

Reply:
Ah, right, so because this validation depends on a question in setup section (type), whenever this validation gets triggered this question also appears in the setup section. 
When the setup section isn't completed we don't allow the rest of the sections to be started, so whenever this validation gets triggered, setup section goes from completed to in_progress and locks and clears all the other sections. Making this questions optional basically prevents that.